pywbem/_subscription_manager.py

```diff
@@ -595,38 +641,46 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True,
 
         Returns:
 
-            :class:`py:list` of :class:`~pywbem.CIMInstance`: The created
-            listener destination instances for the defined listener URLs.
+            :class:`~pywbem.CIMInstance`: The created listener destination
+            instance for the defined listener URL.
 
         Raises:
 
             Exceptions raised by :class:`~pywbem.WBEMConnection`.
+            CIMError(CIM_ERR_ALREADY_EXISTS): A filter with the specified
+              or generated 'Name' property already exists in the server.
+            ValueError: Incorrect input parameter values.
         """
 
         # server_id is validated in _create_...() method.
 
-        # If list, recursively call this function with each list item.
-        if isinstance(listener_urls, list):
-            dest_insts = []
-            for listener_url in listener_urls:
-                new_dest_insts = self.add_listener_destinations(
-                    server_id, listener_url, owned)
-                dest_insts.extend(new_dest_insts)
-            return dest_insts
-
-        # Here, the variable will be a single list item.
-        listener_url = listener_urls
+        if owned:
```

Comment:
This is a question right now. Now that we have explicitly destination_id and name, do we still need the owned parameter for either add_destination or add_filter?  Isn't the ownership clearly defined by the existence of destination_id or name parameters? We do need it for add_subscriptions I agree.

I understand why it seems logical since that clarifies the concept of ownership clearly in the code but it really is redundant and causes an extra set of tests to be required in the code to confirm that the user correctly entered the combination of destination_id, name, and owned.

Reply:
The ownership is clearly an implication of the use of the ID parameter.
However, the issue is that it is not explicit. One of the Python mantras is: "explicit is better than implicit".

Reply:
We discussed this in our call this week and agreed to go with the explicit "owned" parameter even if it is redundant.